processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,67 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = "ul.ul-licitacoes li"
```

Comment:
**suggestion**: Considering that we only use this selector once, and we have many other selectors inside `parse` method, in my opinion having this here is not helpful. If we need to change it because the layout changed, it is very likely that we will have to update the entire spider anyway.